src/utility/localStorage.js

```diff
@@ -0,0 +1,5 @@
+const CART = "CART_ITEMS";
```

Comment:
in utils keep filename same as method name, also method name is wrong since this is global util take CART as a parameter and use it in cart, please check Amruth's PR for same